content/600-about/200-prisma-docs/20-style-guide/07-user-interace-guidelines.mdx

```diff
@@ -81,7 +81,7 @@ Use bold for the names of all UI controls.
 | window                          | In the **Payment information** pop-up window, enter the details for your payment method.                                                                                                                  |
 | screen/tab                      | On the **Data Browser** screen, select a record.                                                                                                                                                          |
 | pane/panel                      | In the **Configuration** panel, click **Network**.                                                                                                                                                        |
-| section                         | Under **Data Proxy**, select **Enabled** from **Static IPs**.                                                                                                                                             |
+| section                         | Under **Accelerate**, select **Enabled** from **Static IPs**.                                                                                                                                             |
```

Comment:
I don't have a strong opinion on this one, but this statement would be incorrect now, as Accelerate doesn't have the option of Static IPs yet.

Reply:
Yup, I had a similar thought but since this is only a reference for writing style, I ignored the technical correctness of this statement.